src/braket/circuits/circuit.py

```diff
@@ -1461,12 +1455,11 @@ def _get_frames_waveforms_from_instrs(
                 for waveform in instruction.operator.pulse_sequence._waveforms.values():
                     _validate_uniqueness(waveforms, waveform)
                     waveforms[waveform.id] = waveform
-            # this will change with full parametric calibration support
             elif isinstance(instruction.operator, Parameterizable):
```

Comment:
```suggestion
            # this will change with full parametric calibration support
            elif isinstance(instruction.operator, Parameterizable):
```